packages/shared/__tests__/stylex-define-consts-test.js

```diff
@@ -0,0 +1,189 @@
+/**
+ * Copyright (c) Meta Platforms, Inc. and affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ * @flow strict
+ */
+
+import styleXDefineConsts from '../src/stylex-define-consts';
+import createHash from '../src/hash';
+import * as messages from '../src/messages';
+
+const themeName = 'TestTheme.stylex.js//buttonTheme';
+const classNamePrefix = 'x';
+
+describe('styleXDefineConsts', () => {
+  test('returns correct structure for basic constants', () => {
+    const constants = {
+      sm: '(min-width: 768px)',
+      md: '(min-width: 1024px)',
+    };
+
+    const [jsOutput, injectableStyles] = styleXDefineConsts(constants, {
+      themeName,
+    });
+
+    expect(jsOutput).toEqual({
+      sm: '(min-width: 768px)',
+      md: '(min-width: 1024px)',
+    });
+
+    expect(injectableStyles).toMatchObject({
+      [classNamePrefix + createHash(`${themeName}.sm`)]: {
+        constKey: 'sm',
+        constVal: '(min-width: 768px)',
+        key: classNamePrefix + createHash(`${themeName}.sm`),
```

Comment:
Use `constKey` for the generated varName.
```suggestion
        constKey: classNamePrefix + createHash(`${themeName}.sm`),
        constVal: '(min-width: 768px)',
```